graph_specs/clingen_data.yaml

```diff
@@ -0,0 +1,9 @@
+# testing graph spec
+graphs:
+  - graph_id: clingen_data
+    graph_name: clingen_data
+    graph_description: 'A fake description for the testing baseline!'
```

Comment:
If you're going to include this yaml file, then the description and url, if provided, should be meaningful.